exampleCourse/questions/demo/cGrader/info.json

```diff
@@ -0,0 +1,17 @@
+{
+    "uuid": "1e1a7e46-1511-49b8-8869-744276a309a0",
+    "title": "Demo: C Grader",
+    "topic": "C",
+    "tags": [
+        "jonatan", "demo", "autograder", "c"
+    ],
+    "type": "v3",
+    "singleVariant": true,
+    "gradingMethod": "External",
+    "externalGradingOptions": {
+        "enabled": true,
+        "image": "prairielearn/grader-c",
+        "timeout": 1000,
```

Comment:
This is in seconds, so I worry that 1000 is rather long to detect that student code is spinning in an infinite loop. Maybe we could shorten this to 30 or something like that?

Reply:
I changed it to 100. The tests themselves have a timeout as well (default to 1 second each), but you're right, it's good to have another external control on the timeout.